app/Http/Controllers/Order/BaseOrderController.php

```diff
@@ -274,40 +274,73 @@ public function sendOrderMail($userid, $orderid, $itemid)
 
     public function getMail($setting, $user, $downloadurl, $invoiceurl, $order, $product, $orderid, $myaccounturl)
     {
+        $value = Product::where('name', $product)->value('type');
         $mail = new \App\Http\Controllers\Common\PhpMailController();
         $mailer = $mail->setMailConfig($setting);
         $templates = new \App\Model\Common\Template();
-        $temp_id = $setting->order_mail;
+        $temp_id = ($value != '4') ? $setting->order_mail : '16';
```

Comment:
don't directly use this 16 what is this?

Reply:
That is the mail template id. Instead of the template name, I just passed the id.
@SarfrazS Did you run the php artisan database: sync?

Reply:
id can change when we run sync, many times. We need to differentiate in a different way